src/legacy-theme/js/colors/light.js

```diff
@@ -0,0 +1,602 @@
+const colors = {
```

Comment:
Could these files be `.ts` with the variable set `as const`? This could mean that the files are checked with TypeScript and we don't have to check-in the generated definition files.

Reply:
Totally! Thanks, will make that change 😄

Reply:
@joshblack took a stab at converting it all to JS, can you give it a look? I don't _totally_ know what I'm doing.